guides/serialization_and_saving.py

```diff
@@ -421,37 +421,109 @@ def from_config(cls, config):
 object that is created from the config.
 The default implementation returns `cls(**config)`.
 
-**Example:**
+**Custom Model Example**
+
+When you subclass a `keras.Model`, the `get_config` should return a dictionary
```

Comment:
I think I want to specify that the config items are explicitly those that are passed in the constructor of the class.

Reply:
That is the default implementation of `from_config`: `return cls(**config)`, and that's how it's often implemented, but the specification here is actually a little more general. The only real requirement is that the result of `get_config` is understood by `from_config`:

```
config = my_obj.get_config()
new_model = type(my_obj).from_config(config)
```
There's no requirement `cls(**config)` to work. IIUC the goal of `from_config` to provide an adapter between `config` and `__init__` so that if they aren't 100% in sync you can fix it (for example if you change __init__ but still want the option to load old configs, or if you want to dispatch to alternate constructors, or modify the object after calling the constructor.)